observing/schedule.py

```diff
@@ -60,6 +61,32 @@ def get_sched():
     return dd, dd2
 
 
+def check_sched(sched):
```

Comment:
rename to `is_conflicted` or something to make it clear that it returns a bool (instead of throwing an exception or something).